mrp_unbuild_cust_alu_analytics/security/mrp_unbuild_cust_alu_analytics_security.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <data noupdate="0">
```

Comment:
Record rules should be always defined in the way admin user can change them:
```suggestion
    <data noupdate="1">
```

If you take a look again at the provided example you could realize that rules are under a `noupdate="1"` section, and group ones are not. See more at https://github.com/solvosci/slv-docker-odoo-prod/wiki/Odoo-SLV-contribution-brief-guidelines#5-security